sdk/communication/Azure.Communication.Calling.Server/src/ConversationClient.cs

```diff
@@ -0,0 +1,546 @@
+        /// <param name="callbackUri">The callback Uri to receive PlayAudio status notifications. </param>
+        /// <param name="operationContext">The operation context. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <returns></returns>
+        public virtual Response<PlayAudioResponse> PlayAudio(string conversationId, Uri audioFileUri, string audioFileId = null , Uri callbackUri = null, string operationContext = null, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ConversationClient)}.{nameof(PlayAudio)}");
+            scope.Start();
+            try
+            {
+                Argument.AssertNotNull(audioFileUri, nameof(audioFileUri));
+
+                // Currently looping media is not supported for out-call scenarios, thus setting it to false.
+                return RestClient.PlayAudio(conversationId, audioFileUri.AbsoluteUri, false, operationContext, audioFileId, callbackUri?.AbsoluteUri, cancellationToken);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Add participant to the call.
+        /// </summary>
+        /// <param name="conversationId"> The conversation id that can be a group id or a encoded conversation url retrieve from client. </param>
+        /// <param name="participant"> The identity of participant to be added to the call. </param>
+        /// <param name="callbackUri">The callback uri to receive the notification.</param>
+        /// <param name="alternateCallerId">The phone number to use when adding a pstn participant.</param>
+        /// <param name="operationContext">The operation context.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        public virtual Response AddParticipant(string conversationId, CommunicationIdentifier participant, Uri callbackUri, string alternateCallerId = default, string operationContext = default, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ConversationClient)}.{nameof(AddParticipant)}");
+            scope.Start();
+            try
+            {
+                Argument.AssertNotNull(participant, nameof(participant));
+
+                var participantsInternal = new List<CommunicationIdentifierModel> { CommunicationIdentifierSerializer.Serialize(participant) };
+                var alternateCallerIdInternal = string.IsNullOrEmpty(alternateCallerId) ? null : new PhoneNumberIdentifierModel(alternateCallerId);
+                return RestClient.InviteParticipants(conversationId, participantsInternal, alternateCallerIdInternal, operationContext, callbackUri?.AbsoluteUri, cancellationToken);
```

Comment:
nit: let's break these long lines into multiple:
```C#
return RestClient.InviteParticipants(
    conversationId,
    participantsInternal,
    ....
```

Reply:
This is addressed.